pkg/kubelet/client/kubelet_client.go

```diff
@@ -36,6 +36,9 @@ type KubeletClientConfig struct {
 	Port        uint
 	EnableHttps bool
 
+	// PreferredAddressType - used to select an address from Node.NodeStatus.Addresses
+	PreferredAddressType []string
```

Comment:
``PreferredAddressType`` suggests a single value rather than a preference-ordered slice.  Consider using a more descriptive name (e.g. AddressTypePrecedence).